server/tests/unit/__init__.py

```diff
@@ -149,6 +151,26 @@ def _before_adding_routes(app, app_config):
         if app_config.server_config.app__debug:
             CORS(app, supports_credentials=True)
 
+    @staticmethod
+    def _extract_base_url_and_dataset_for_api_calls(app_config: AppConfig):
```

Comment:
convenience method for generating the url for the developer to use to access the front end since it is dependent on the name of the dataset being used (and the `CXG_CLIENT_PORT`, if set)